### DISCUSSION TITLE
Does the Emporia Vue support bluetooth BLE?

### DISCUSSION THREAD

**Discussion by ripvega** [emporia-vue-local/esphome#47] [Q&A] [answered]
I have 5 Emporia Vue converted to this fantastic ESPHome project.

With Home Assistant 2022.10 the [bluetooth_proxy integration](https://esphome.github.io/bluetooth-proxies/) was added.

Making use of this in the existing Emporia Vues would be a great addition and a reduction in additional hardware.

Any reason why this won't work?

It's simply a case of adding `bluetooth_proxy: `to your ESPHome Emporia config and then Home Assistant should auto-discover it.

Happy to give it a try but thought I'd check here in case it's a no-no.

**Comment by flaviut** [comment]
Go try it! I wouldn't expect it to break anything. And if it does break something, you can always remove it from your panel and reflash over serial.

**Comment by DelusionalAI** [comment, marked as answer]
I gave it a go and then it dropped from the wifi... like it was having issues with the antenna being used for BLE or Wifi. I haven't had a chance to rip it out of my panel yet to re-flash it.

**Comment by whoiswes** [comment]
Yeah, I tried this too and had the same outcome.  Needed a reflash to get back online.  However, this device is my first foray into the ESPHome world so it may have been and ID10T error in my case.

**Comment by TommySharpNZ** [comment]
I've added the BLE proxy to my ESPHome config and it seems to still be connected to the wifi... Not entirely sure how I can test to ensure BLE is working though...

**Comment by ashman5555** [comment]
I just tried again with Home Assistant 2022.12.0 and ESPHome 2022.11.5.

No luck. Wifi is still broken.

**Comment by TommySharpNZ** [comment]
Could it be something in your ESPHome code? Do you want to share it?

**Comment by PanicRide** [comment]
`bluetooth_proxy` uses a lot of the limited resources on the ESP32. It's possible that there just aren't enough resources to run both that and the emporia stuff.

ESP32 hardware is very cheap, so I personally wouldn't risk trying to run it on the emporia and have to remove it from the panel for manual flashing if something goes wrong.

**Reply by TommySharpNZ** [reply]
Good point... I have been a bit dubious about the accuracy of my Empria and it sometimes dropping or resetting my daily counters... I wonder if me running the BLE Proxy is causing this... Have disabled for now...

Am also considering just using an ESP32 stick like these to do my BLE Proxies...
https://www.tindie.com/products/lilygo/lilygo-t-dongle-s3-esp32-s3-development-board/
